Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -605,6 +593,37 @@ def verify_bintray_published(formulae_names)
     end
   end
 
+  def check_publication(f, url, max_retries, retry_count)
```

Comment:
I'd call it `check_<something>_status` where `<something>` could be `bottle`

Reply:
What about `is_bottle_published` (it returns a boolean)?

Reply:
I expect someone from upstream to chime in. I just expressed my opinion :)